account/src/msg/account-err.ts

```diff
@@ -0,0 +1,50 @@
+import { status } from '@grpc/grpc-js';
+//import { IGrpcErr } from './account-grpc-interface';
+import { RpcException } from '@nestjs/microservices';
+import { ArgumentsHost, Catch, ExceptionFilter, RpcExceptionFilter } from "@nestjs/common";
+import { BaseExceptionFilter } from '@nestjs/core';
+import { Request, Response } from 'express';
+import { callbackify } from 'util';
+import { Observable, of, throwError } from 'rxjs';
+
+
+export enum codes {
+  UNKNOWN = status.UNKNOWN,
+  INVALID_ARGS = status.INVALID_ARGUMENT,
+  NOT_FOUND = status.NOT_FOUND,
+  ALREADY_EXISTS = status.ALREADY_EXISTS,
+  PERMISSION_DENIED = status.PERMISSION_DENIED,
+  ABORTED = status.ABORTED,
+  UNIMPLEMENTED = status.UNIMPLEMENTED,
+  UNAUTHENTICATED = status.UNAUTHENTICATED,
+}
+
+export enum messages {
+  USER_MADE = 'user created',
```

Comment:
хороший тон использовать одинаковый формат и формулировку для ключа и значение в enum
```suggestion
  USER_CREATED = 'USER_CREATED',
```